src/Layout/components/Header.tsx

```diff
@@ -35,11 +39,11 @@ const Header = () => {
               <span onClick={handleLogout} className="cursor-pointer">
                 Logout
               </span>
-              <NavLink to="/signin">Log in</NavLink>
+              {userLoggedIn ? "" : <NavLink to="/signin">Log in</NavLink>}
             </>
           </div>
           <div className="bottom flex gap-4 justify-end">
-            <NavLink to="/">Home</NavLink>
+            {userLoggedIn ? "" : <NavLink to="/">Home</NavLink>}
```

Comment:
Use the && shortciruit operation

`    {!userLoggedIn && <NavLink to="/">Home</NavLink>}
`

Reply:
Its been resolved